src_assets/common/assets/web/config.html

```diff
@@ -342,10 +342,7 @@ <h1 class="my-4">{{ $t('config.configuration') }}</h1>
         this.$forceUpdate()
       },
       serialize() {
-        let config = JSON.parse(JSON.stringify(this.config));
-        config.global_prep_cmd = JSON.stringify(config.global_prep_cmd);
```

Comment:
You might have to stringify these at line 360 for comparison, so that the default values are not saved

Reply:
Default values are not saved.

This is a test config where I only changed 3 options.

```txt
min_fps_factor = 3
notify_pre_releases = enabled
sunshine_name = CMake-Release
```

Reply:
Just to be sure, save both of those values with non-default configs and then undo to get default configs.

Reply:
`global_prep_cmd` works fine without stingify, but the dd one doesn't 🤷‍♂️

Reply:
Your suggestion from discord works, thanks!